fixed-requirements.txt

```diff
@@ -10,6 +10,7 @@ cryptography==2.8
 eventlet==0.25.1
 flex==6.14.0
 gitpython==2.1.15
+git+https://github.com/StackStorm/st2-rbac-backend.git@master#egg=st2-rbac-backend
```

Comment:
I tested the dev packages from this PR and indeed rbac package wasn't picked up.

Let's try to include the rbac in `st2common/in-requirements.txt` instead of this file and generate the requirements.txt with `make requirements`.

That should help.

Reply:
OK this looks good now.